crates/astria-conductor/src/executor/mod.rs

```diff
@@ -146,20 +146,18 @@ impl Handle<StateIsInit> {
     #[instrument(skip_all, err)]
     pub(crate) async fn send_firm_block(
         self,
-        block: ReconstructedBlock,
+        block: Box<ReconstructedBlock>,
     ) -> Result<(), FirmSendError> {
         let sender = self.firm_blocks.as_ref().ok_or(FirmSendError::NotSet)?;
-        sender.send(block).await?;
-        Ok(())
+        Ok(sender.send(block).await?)
     }
 
     pub(crate) fn try_send_firm_block(
         &self,
-        block: ReconstructedBlock,
+        block: Box<ReconstructedBlock>,
```

Comment:
Similar to `fn try_send_firm_block`:

```suggestion
        block: impl Into<Box<ReconstructedBlock>>,
```

Reply:
Thanks for these catches and suggestions! Made the requested changes in d2338a04.